apps/website/src/components/OpenInCodePen.astro

```diff
@@ -13,6 +13,8 @@ const css = outdent`
     padding: 1rem;
   }
 
+  ${exampleStyle}
```

Comment:
I'm thinking it would be nice if the entirety of the "CSS" pane only contained the demo CSS. 🤔

Reply:
We can remove the box-sizing style, and move the rest into inline styles. The body styles can be moved into HTML and the app-wrapper styles can be moved into JSX.

![image](https://github.com/iTwin/iTwinUI/assets/9084735/d03781b8-5e5e-4242-b0f3-0136d78b1b82)

Reply:
Will do.

Reply:
Most recent commit resolves this.

Reply:
[`outdent`](https://www.npmjs.com/package/outdent) is not necessary if you don't have any indented code. You can use simple strings

```js
const css = code.css.trim();
const html = `<body style='margin: 0'></body>`
```

Reply:
`outdent` has been removed from these css and html values.